### PR TITLE
Adding standard size to calendar strip (fixing basic example)

### PR DESCRIPTION
**Pull request by zosman1**
I've been trying to implement this repository for some time and have not been able to get the `Simple "out of the box" Example` working. 
After some investigation this appears to be because the view that contains this component has no inherent size. I added a "default" style to the view that includes size and some padding, this can be easily overwritten using the style prop for this component. 

**tl;dr**
Simple "out of the box" Example not working -> working

### CODE REVIEW

**Comment by peacechen** [comment]
That's correct, CalendarStrip does not set a fixed height because it sizes to its parent container.  The parent can set a fixed height, or use `flex: 1` to dynamically size.  The example project sets the style prop:
https://github.com/BugiDev/react-native-calendar-strip/blob/master/example/example/App.js#L16

I would not recommend setting a fixed height in this library because different apps create different size CalendarStrips.  Hard-coding a height may be convenient but could actually be more confusing because a user setting a width (but not height) expects it to size dynamically, but with a default fixed height, it could get truncated.  Have you tried setting the container style to `flex: 1`?  Perhaps that should be the default.  If so, it would be better placed under defaultProps to make the setting easier to locate.

**Comment by zosman1** [comment]
simulator screen shot - iphone 6 - 2018-10-16 at 15 03 28
So this is using an edit of the repo setting the container style to flex 1 by default, with the app just rendering this component inside a view with flex 1.
i'd like to make it look a bit better by default, do you have any recommendations?

**Comment by peacechen** [comment]
I've updated the "out of the box" example in the Readme.  Thanks for bringing this to our attention.  It's important to help new users get started quickly.
